### PR TITLE
Delay latejoin by X minutes (to be defined)

### PR DESCRIPTION
**Pull request by ShiftyRail**
## Reasoning

The point is to avoid people latejoinings 2.5 seconds into a round to secure their favourite toy.
30 minutes is way too much, but I'm not sure what a reasonable delay would be.

It is toggleable by admins, and it's also automatically off on deadpop (<15 people).

Also this is not tested yet

### CODE REVIEW

**Comment by MusgardTiger** [comment]
Delaying latejoining by 10 minutes and removing latejoin heads would be good I think. 30 minutes is actually too much.

**Comment by ShiftyRail** [comment]
Please post on PR the delays you wish

@MusgardTiger No heads roundstart should not mean no heads at all.

**Comment by MusgardTiger** [comment]
We can allow for late-join HoP to ensure there'd be at least one other guy with access to give more access if necessary. Removing latejoin heads would encourage people to play those roles more I think.

Edit: In my brain late join and late join head role situation is intertwined but I don't know if you had the same thing in mind while making this so I can drop this no late join heads discussion for a PR that is relevant if this one isn't.

**Comment by Irradiation** [comment]
Just like, secure the spare and self-promote to Head status and hand out Head status. I'm sure the spare holder is a responsible spessmen that has the best interests of the station at heart and totally won't just give himself all-access and go braindead/die somewhere unreachable. Maybe he'll grant all-access to a few people (Read: greyshirts) if you're pseudo lucky.

**Comment by Irradiation** [comment]
On a serious note, I'd say a 7 minute delay sounds good enough @ShiftyRail .

**Comment by aceedex** [comment]
As somsone said:
What if Im set to go back to lobby if preferences are not avalible

**Comment by Irradiation** [comment]
I'd say slap a 1 minute delay on job-preference-filled-lobby-returners to they don't get punished too much. _Hopefully_ people don't use this shortened delay to do exactly our current problem, but with a  minute dlay instead of .001 seconds.

**Comment by DamianX** [comment]
>The point is to avoid people latejoinings 2.5 seconds into a round to secure their favourite toy.

I'm a bit thick, can you elaborate?

**Comment by iFlashYou** [comment]
Don't forget the "joins 1 second into the round to avoid being antag target"

**Comment by gbasood** [comment]
Would it be possible to have the latejoin be a configurable time instead? A choice of setting it to 30 minutes or instant is a huge jump.

**Comment by ghost** [comment]
What's the fucking point?
Isn't the pop already low enough, do you want to lower it even more by fucking over anyone that connects shortly after roundstart? And what do you even gain from it besides an even lower pop?

**Comment by strawberrytan** [comment]
I believe the point is to weaken people's ability to join after roundstart and reviewing the manifest

I wouldn't put it above 5 minutes so it keeps parity with pre round setup and I don't see why you'd kill the ability to be a head after roundstart, what about left b. hind?

**Comment by strawberrytan** [comment]
Actually let me rephrase that, I wouldn't put it above 15 minutes and suggest 5 as a starting point

**Comment by Fruktlimpa** [comment]
30 minutes is huge and too long in my opinion. 
IF we are going to do this we should definitely do a test run of it and then review it again later, and like Strawberrytan said 5 minutes is a good starting point and not above 15.

**Comment by GreenHairedAssistantRickysSmellyDick** [comment]
Literally who wants this? Does any player of the server want this change? It's literally just making it harder to play the game. Sometimes I want to see what jobs need filling before joining, sometimes I want to look at the manifest and think "is this a good manifest or is it full of shitters" before I decide to play. Now you want to force me to wait an absurd amount of time on top of the 5 minute lobby time, just to play?

Is this just another part of MustardTiger's plan to add a bunch of policies that do nothing but make the server actively worse, as he's the only person supporting this bullshit PR. (Check the forums and the discord, yikes). No I will not be supporting this github pull request and urge everyone else to also downvote this potential change. In a dying server, the thing we do not need is features that restrict good players from actually playing. It is statistically known shitters don't late join, they always ready up for their antag roll, this change exists to create a station spawned with shitters and delaying the time until decent ordinary players can play. Even 3 minutes will mean the spare is gone every shift.

**Comment by ShiftyRail** [comment]
The returns show that it's not an adequate solution.
